modules/cloud_logs/variables.tf

```diff
@@ -0,0 +1,132 @@
+variable "region" {
+  description = "The region where observability resources are created."
+  type        = string
+  default     = "eu-es"
+}
+
+variable "resource_group_id" {
+  type        = string
+  description = "The id of the IBM Cloud resource group where the instance(s) will be created."
+  default     = null
```

Comment:
is this not required?

Reply:
![Screenshot 2024-07-31 at 10 12 17 AM](https://github.com/user-attachments/assets/a0847f3a-554d-4e65-b384-b41425a904fb)

Cloud logs instance is scoped to the resource group.

Reply:
right, so it is required. So it should not have a default value of `null`. It should have no default value